engine/src/flutter/lib/ui/semantics.dart

```diff
@@ -362,6 +362,72 @@ enum SemanticsRole {
 
   /// The main display for a tab.
   tabPanel,
+
+  /// An input field for users to enter search terms.
+  searchBox,
+
+  /// A control that used for dragging across content.
+  ///
+  /// e.g, the drag handle of [ReorderableList].
+  dragHandle,
+
+  /// A control to cycle through content on tap.
+  ///
+  /// e.g, the next and previous month button of a [CalendarDatePicker].
+  spinButton,
+
+  /// A input field with a dropdown list box attached.
+  ///
+  /// e.g, a [DropDownMenu]
+  comboBox,
+
+  /// Contains a list of [menu].
+  ///
+  /// e.g, a [MenuBar].
+  menuBar,
+
+  /// A button that opens a dropdown that contains multiple [menuItem]s.
+  ///
+  /// e.g, a [MenuAnchor] or [DropDownButton].
+  menu,
+
+  /// Items in a dropdown created by [menu] or [comboBox].
+  menuItem,
+
+  /// A container to display multiple [listItem] in vertical or horizontal
+  /// layout.
+  ///
+  /// e.g, a [LisView] or [Column].
+  list,
+
+  /// An item in a [list].
+  listItem,
+
+  /// An area that resembles a form.
+  form,
+
+  /// A pop up when hover over a component to provide contextual explanation.
+  tooltip,
+
+  /// A control that controls the scrolling of content within a viewing
+  /// area.
+  scrollBar,
```

Comment:
Do we publish scrollbars to the semantics? I'm not sure how this would be mapped onto the framework in a way that would actually integrate with screen readers 🤔

Reply:
this is for custom scrollbar, not the native scroll bar.
https://api.flutter.dev/flutter/material/Scrollbar-class.html

Reply:
Yeah, I just don't know what will happen if this gets into our semantics tree. Will we crash? :)

Reply:
interestingly in native html, setting the scrollbar role will cause voiceover to ignore that element. our scrollbar doesn't seem to contribute to semantics it seem. I will remove this enum